editors/sc-ide/core/doc_manager.cpp

```diff
@@ -773,7 +773,7 @@ void DocumentManager::handleNewDocScRequest( const QString & data )
             Q_EMIT( opened(document, 0, 0) );
         }
     } catch (std::exception const& e) {
-        qWarning() << "DocumentManager::" << __func__ << ": could not handle request:" << e.what();
+        qWarning() << "DocumentManager::" << __FUNCTION__ << ": could not handle request:" << e.what();
```

Comment:
can you explain this? according to stackoverflow, `__FUNCTION__` is [nonstandard](https://stackoverflow.com/questions/597078/file-line-and-function-usage-in-c#597081).